common/locale.go

```diff
@@ -0,0 +1,38 @@
+package common
+
+import (
+	"context"
+	"net/http"
+	"strings"
+)
+
+// LocaleCode the locale code to use to determine the language to use
+const LocaleCode = ContextKey("LocaleCode")
```

Comment:
I would consider making a Lang const:

```
const LangEN = "en"
const LangCY = "cy"
...
type Lang string
```
Minor comment but they are constants and IMO it's a bit nicer dealing with a type or constant that a string.

Reply:
Absolutely, shall replace the magic strings for the constants, also beneficial if we replace the localeCode to 'en_GB', 'en-GB', 'EN' or something different to "en".